### ISSUE TITLE
Kitti Validation Split

### ISSUE THREAD

**Issue by maaft** [fangchangma/sparse-to-dense.pytorch#20]
Hi,

in your paper you talk about "[...] a random subset of 3200 images from the test sequences [...]" for validation on the Kitti Dataset.

A fresh checkout of this repo will validate the network against the full 40k validation images though, which results in very different results as reported in your paper (i.e. RMSE of 4.7m vs 6.2m).

Could you elaborate on which 3200 images you took specifically?

**Comment by fangchangma** [comment]
Hi. I also realized that the rgb-based depth prediction accuracy is higher in this pytorch version. I suspect that this is because the PyTorch Imagenet-pretrained network is somehow "better" than the torch version. In particular, for depth completion tasks (with sparse depth samples), the performance evaluated on the entire test sequence is close to the paper, if I recall correctly.
